docs/website/docs/building-from-source/ios.md

```diff
@@ -0,0 +1,130 @@
+# Cross-compilation for iOS
+
+Cross-compilation for iOS consists of the two steps below.
+
+* On the macOS host, build the IREE compiler.  We can run it to create
+  IREE modules.
+* Build the IREE runtime on the macOS host for iOS devices and the
+  simulator.  We can then run the IREE module on the simulator.
+  
+## Prerequisites
+
+### Install Xcode and iOS SDK
+
+For cross-compilation, you need Xcode. It comes with the SDKs for iOS
+devices and the simulator, as well as the `simctl` tool for
+controlling the simulator from the command line.
+
+### Host environment setup
+
+On your host platform, you should already be able to build IREE from
+source.  Please make sure you've gone through the steps in [getting
+started](./getting-started.md).
+
+## Configure and Build
+
+### Build the IREE Compiler for the Host
+
+Build and install on your macOS host:
+
+``` shell
+cmake -S . -B ../iree-build/ -GNinja \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DCMAKE_INSTALL_PREFIX=../iree-build/install
+  
+cmake --build ../iree-build/ --target install
+```
+
+## Cross-compile the IREE Runtime for iOS
+
+Build the runtime for the iOS Simulator.
+
+```shell
+cmake -S . -B ../build-ios-sim -GNinja \
+  -DCMAKE_SYSTEM_NAME=iOS \
+  -DCMAKE_OSX_SYSROOT=$(xcodebuild -version -sdk iphonesimulator Path) \
+  -DCMAKE_OSX_ARCHITECTURES=arm64 \
+  -DCMAKE_SYSTEM_PROCESSOR=arm64 \
+  -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
+  -DCMAKE_IOS_INSTALL_COMBINED=YES \
+  -DIREE_HOST_BINARY_ROOT=$PWD/../iree-build/install \
+  -DCMAKE_INSTALL_PREFIX=../build-ios-sim/install \
+  -DIREE_BUILD_COMPILER=OFF -DIREE_BUILD_TESTS=OFF -DIREE_BUILD_SAMPLES=OFF
```

Comment:
Are `-DIREE_BUILD_TESTS=OFF -DIREE_BUILD_SAMPLES=OFF` both required right now for a successful configure/build? We should be able to support both.

Reply:
No. They are not required. I just checked. Should I remove these two OFF flags from this command-line?

Reply:
Yeah, I'd omit them unless they're strictly required. We want these instructions to be as concise as possible.

Reply:
Done